### DISCUSSION TITLE
Completely disable original macOS bar

### DISCUSSION THREAD

**Discussion by thanhhafmcvn** [FelixKratz/SketchyBar#229] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/83865348/182610055-d474003a-68bf-41cb-8a0f-263fa0ffdf09.png)
How can I completely hide the original bar on macOS, I do not want it appear anymore

**Comment by FelixKratz** [comment, marked as answer]
There are some maximal invasive techniques that would completely kill the original bar, i.e. using yabai v6.0.2+:
```bash
yabai -m config menubar_opacity 0.0
```

You can also draw sketchybar on top of the original bar with the `topmost` bar setting:
```bash
sketchybar --bar topmost=on
```

**Reply by thanhhafmcvn** [reply]
Thanks for you supporting. it works and I feel better. Have a great day!

**Reply by jacohend** [reply]
I can't seem to get this to work on Ventura, can anyone confirm?

**Reply by Muneeb-Mohd** [reply]
Thanks bro

**Reply by FelixKratz** [reply]
Updated the reply with a new way to completely disable the original menubar using yabai
